### PR TITLE
Add the timestamp of report generation and version

### PR DESCRIPTION
**Pull request by daltonkell**
Reports (all formats) now include the version of the Compliance Checker
as well as the date the report was run.

### CODE REVIEW

File: compliance_checker/suite.py [hunk 1]
```diff
@@ -584,6 +584,10 @@ def standard_output(self, ds, limit, check_name, groups):
         print("\n")
         print("-" * width)
         print("{:^{width}}".format("IOOS Compliance Checker Report", width=width))
+        print("{:^{width}}".format("Version {}".format(__version__), width=width))
+        print("{:^{width}}".format("Report generated {}".format(
+            datetime.now(timezone.utc).strftime("%Y-%m-%dT%H:%M:%SZ")
```
Comment (ocefpaf) [inline compliance_checker/suite.py:589]: Came here to [paste this](https://blog.ganssle.io/articles/2019/11/utcnow.html) but I'm glad you probably already know about it!

**Comment by daltonkell** [comment]
Actually @ocefpaf this is a great chance to ask -- how do we get the `pre-commit` tests to pass? I'm not familiar with them, but I looked at the build logs and they're quite verbose.

**Comment by ocefpaf** [comment]
> Actually @ocefpaf this is a great chance to ask -- how do we get the `pre-commit` tests to pass?

Merge #756 :smile: 

> I'm not familiar with them, but I looked at the build logs and they're quite verbose.

In the GH action they show all the offending cases but if you run them locally most of them are automatically fixed. Just install pre-commit in your dev env and run `pre-commit run --all-files`. If you it to run with every commit, recommended but annoying, you have to run `pre-commit install`.

File: compliance_checker/suite.py [hunk 2]
```diff
@@ -584,6 +584,10 @@ def standard_output(self, ds, limit, check_name, groups):
         print("\n")
         print("-" * width)
         print("{:^{width}}".format("IOOS Compliance Checker Report", width=width))
```
Comment (benjwadams) [inline compliance_checker/suite.py:586]: I realize this was legacy code included with the project, but I'm not really a fan of these repeated print functions and mutliple calls to format for centering text in a plaintext table.   For things like f-strings, multiple levels could even be an injection hazard with user supplied strings.

  Could you combine the text to be centered into a single multiline string and then call `str.center(width)` instead?

Comment (daltonkell) [reply compliance_checker/suite.py]: Should be fixed to satisfaction with latest commit

**Pull request merged**